File: go/chapter6/people.go
package main

import "fmt"

func main() {
	people := map[string]map[string]interface{}{
		"Jack": {
			"firstName": "Jack",
			"lastName":  "Doe",
			"age":       10,
			"city":      "Freetown",
		},
		"Kate": {
			"firstName": "Kathrine",
			"lastName":  "Smith",
			"age":       13,
			"city":      "Detroit",
		},
	}

	for key, value := range people {
		fmt.Printf("%s\n", key)
		for k, v := range value {
			fmt.Printf("%s: %v\n", k, v)
		}
		fmt.Println()
	}
}

File: python/chapter6/people.py
people = {
    "Jack": {
        "firstName": "Jack",
        "lastName": "Doe",
        "age": 10,
        "city": "Freetown",
    },
    "Kate": {
        "firstName": "Kathrine",
        "lastName": "Smith",
        "age": 13,
        "city": "Detroit",
    },
}

for key, value in people.items():
    print(f"{key}")

    for k, v in value.items():
        print(f"{k}: {v}")

    print()
